pkg/metrics/pq.go

```diff
@@ -182,6 +191,12 @@ type Block struct {
 	MissingIDs     []int64
 	Messages       []string
 	FeeFrac        uint64
+	Transactions   []Transaction
+}
+
+type Transaction struct {
+	Hash    []byte
```

Comment:
pp: how about renaming this field and the the DB field to: `sha256_hash` ?

Reply:
This is redundant, as that is what tendermint uses by default. That is why I used `Hash`. Tendermint could have done it on their side, but it does not for whatever reason.

Reply:
It is not redundant to me as ppl seem to integrate on the DB level. There is no information otherwise which hash algo is used.

Reply:
I will document this in the code then.
I think we need to surface the documentation about transaction hashes somewhere in weave docs too.
I actually didn't know where the transaction hashes are coming from before I had to approach this, so intuitively I've been looking for tendermint to provide those for me as it does allow to query by them.